Source/EventFlow/Snapshots/SnapshotMetadata.cs

```diff
@@ -84,5 +85,19 @@ public int SnapshotVersion
             get { return GetMetadataValue(SnapshotMetadataKeys.SnapshotVersion, int.Parse); }
             set { Add(SnapshotMetadataKeys.SnapshotVersion, value.ToString(CultureInfo.InvariantCulture)); }
         }
+
+        [JsonIgnore]
+        public IReadOnlyCollection<ISourceId> PreviousSourceIds
+        {
+            get { return GetMetadataValue(SnapshotMetadataKeys.PreviousSourceIds, (json) => 
+                string.IsNullOrWhiteSpace(json) ? 
+                    Enumerable.Empty<ISourceId>() : 
```

Comment:
Add a static readonly member that's an empty `IReadOnlyCollection<ISourceId>` and return that instead of doing a `ToList()` on an empty enumerable